### PR TITLE
Add support for ZyXEL IES-5005 and IES-5106 DSLAMs

### PR DESCRIPTION
**Pull request by vitalisator**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Jellyfrog** [comment]
Please remove "WIP" from the title when your pull request is ready

**Comment by vitalisator** [comment]
> Please remove "WIP" from the title when your pull request is ready

I still waiting for merge of #10789 , then we need a rebase here.

File: includes/polling/os/ies5000.inc.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+# ies 5106
+if ($device['sysObjectID'] == '.1.3.6.1.4.1.890.1.5.13.10') {
+    $r_temp = snmp_get_multi_oid($device, ['.1.3.6.1.4.1.890.1.5.13.5.6.2.1.3.0', '.1.3.6.1.4.1.8886.6.1.1.1.2.0', '.1.3.6.1.4.1.8886.6.1.1.1.14.0'], '-OQUn');
+    $serial   = $r_temp['.1.3.6.1.4.1.890.1.5.13.5.6.2.1.3.0'];
```
Comment (PipoCanaja) [inline includes/polling/os/ies5000.inc.php]: Hello,
You poll a few OIDs but only use one of them ? If you don't need the others, please avoid polling them. 
Thx

Comment (vitalisator) [reply includes/polling/os/ies5000.inc.php]: fixed, thx for hint.

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-58-release-changelog-november-2019/10224/1

**Pull request merged**